<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QHeaderView::section:checked
{
    background-color: red;
}

/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
      <column>
       <property name="text">
        <string>歌曲</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Url</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="playing">
      <property name="statusTip">
       <string>正在播放</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="shengyintiaojie">
      <property name="statusTip">
       <string>音量调节</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="jindutiaojie">
      <property name="statusTip">
       <string>进度调节(暂不可用)</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionImport_song"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionPause"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="pattern">
    <property name="title">
     <string>播放模式</string>
    </property>
    <addaction name="actionDanquxunhuang"/>
    <addaction name="actionShunxubofang"/>
    <addaction name="actionRadom"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout_me"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="pattern"/>
   <addaction name="about"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_song"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="separator"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_me"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_song">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/new/prefix1/images/7.png</normaloff>:/new/prefix1/images/7.png</iconset>
   </property>
   <property name="text">
    <string>import song</string>
   </property>
   <property name="statusTip">
    <string>导入歌曲</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/new/prefix1/images/2.png</normaloff>:/new/prefix1/images/2.png</iconset>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
   <property name="statusTip">
    <string>上一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/new/prefix1/images/5.png</normaloff>:/new/prefix1/images/5.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="statusTip">
    <string>下一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/new/prefix1/images/4.png</normaloff>:/new/prefix1/images/4.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="statusTip">
    <string>暂停/播放</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionDanquxunhuang">
   <property name="text">
    <string>repeat</string>
   </property>
   <property name="statusTip">
    <string>单曲循环</string>
   </property>
  </action>
  <action name="actionShunxubofang">
   <property name="text">
    <string>order</string>
   </property>
   <property name="statusTip">
    <string>顺序播放</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="icon">
    <iconset>
     <normaloff>../../库/图片/132905kdid4nu4pizpujd6.jpg</normaloff>../../库/图片/132905kdid4nu4pizpujd6.jpg</iconset>
   </property>
   <property name="text">
    <string>about me</string>
   </property>
   <property name="statusTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionRadom">
   <property name="text">
    <string>radom</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about music</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="music.qrc"/>
 </resources>
 <connections/>
</ui>
